client/components/Button.tsx

```diff
@@ -8,6 +8,7 @@ export interface ButtonProps {
   type?: 'submit' | 'reset' | 'button';
   disabled?: boolean;
   children: React.ReactNode;
+  classes?: string;
```

Comment:
This leaves us open to having divergent button styles across the app and circumvents the component's consistency 

It doesn't look like this prop is used in this PR, can we remove this?

Reply:
Removed 'classes'.